### PR TITLE
Add a bit more info to the ckan docs

### PR DESCRIPTION
**Pull request by hannako**
Minor update to ckan docs

https://trello.com/c/XriMyZkM/1198-update-ckan-docs-on-how-to-run-ckan-commands-on-the-machines

### CODE REVIEW

File: source/manual/data-gov-uk-supporting-ckan.html.md [hunk 1]
```diff
@@ -44,14 +44,21 @@ You can SSH onto these machines in [same way as all other GOV.UK AWS application
...
+
+Most of the commands to interact with [CKAN] use the `ckan` CLI.  Some of these
 commands take a path to the config file with the `-c` option, which is located at `/var/ckan/ckan.ini`
 in our deployments.
 
-On GOV.UK servers `ckan` should be run with:
+Once connected to the `ckan` machine, the commands should be run from within the var directory as follows:
 
 ```
+cd ../../var
```
Comment (ChrisBAshton) [inline source/manual/data-gov-uk-supporting-ckan.html.md:61]: Could you make this an absolute path? (Run `pwd` to find out what dir you're in after running the `cd ../../var`, and paste that here instead)

Comment (hannako) [reply source/manual/data-gov-uk-supporting-ckan.html.md:61]: sure one sec

File: source/manual/data-gov-uk-supporting-ckan.html.md [hunk 2]
```diff
@@ -44,15 +44,21 @@ You can SSH onto these machines in [same way as all other GOV.UK AWS application
...
+Most of the commands to interact with [CKAN] use the `ckan` CLI.  Some of these
 commands take a path to the config file with the `-c` option, which is located at `/var/ckan/ckan.ini`
 in our deployments.
 
-On GOV.UK servers `ckan` should be run with:
+Once connected to the `ckan` machine, the commands should be run from within the `/var` directory as follows:
 
 ```
-sudo -u deploy govuk_setenv ckan /var/apps/ckan/venv3/bin/ckan [COMMAND]
+user@ec2-integration-blue-ckan-ip-1-2-3:/var$ sudo -u deploy govuk_setenv ckan /var/apps/ckan/venv3/bin/ckan [COMMAND]
```
Comment (ChrisBAshton) [inline source/manual/data-gov-uk-supporting-ckan.html.md:61]:
```suggestion
sudo -u deploy govuk_setenv ckan /var/apps/ckan/venv3/bin/ckan [COMMAND]
```

File: source/manual/data-gov-uk-supporting-ckan.html.md (hunk 1 above)
Comment (ChrisBAshton) [reply source/manual/data-gov-uk-supporting-ckan.html.md]: Thanks, looks better 👍

**Review by ChrisBAshton — APPROVED** [review]

**Pull request merged**